gnat2goto/driver/tree_walk.adb

```diff
@@ -1198,6 +1222,53 @@ package body Tree_Walk is
       return Ret;
    end Do_Enumeration_Definition;
 
+   function Do_Attribute_Pos_Val (N : Node_Id) return Irep is
```

Comment:
would be nice to have a comment why attribute pos and val are implemented equivalently.